contents/surveys/index.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Surveys
+subtitle: <span class="text-red font-semibold bg-yellow/10">Gauge product market fit</span>, measure Net Promoter Score (NPS), schedule user interviews and more
+productHero: {
+  video: ../images/tutorials/survey/surveys_hero_small.mp4
+}
+template: product
+productFeatures: [
+    {
+        icon: 'Graph',
+        title: 'Data visualization for results',
+        description: 'Qualitative data? Quantative data? Just get the best of both.'
+    },
+    {
+
+        icon: 'Bell',
+        title: 'Slack notifications',
+        description: 'Because having to paste new responses in one-by-one is a PITA.'
```

Comment:
I don't think non-British people will understand what PITA means sadly 😢 
```suggestion
        description: 'Because having to paste new responses in one-by-one is tedious.'
```